src/protocol/verification.md

```diff
@@ -1,31 +1,19 @@
 # Verification
 
-A `Verifier`` receives the following from the `User`:
+To prove data provenance to a third-party `Verifier`, the `User` provides them the following:
+- [`Session Header`](/protocol/notarization/signing.md) signed by the `Notary`
+- `opening` to the plaintext commitment
+- `TLS-specific data` which uniquely identifies the server
+- `identity` of the server
 
-// TODO will explain each
+The `Verifier` performs the following verification steps:
+- verifies that the `opening` corresponds to the commitment in the `Session Header`
+- verifies that the `TLS-specific data` corresponds to the commitment in the `Session Header`
```

Comment:
I feel like it would add more value to be a little more specific here about the commitments, like for example in the case of `TLS-specific data` I assume you are talking about the commitment to the `HandshakeData`.

Or is this a level of detail which you want to deal with in `verification_steps.md` only?

Reply:
We wanted to make this section as light on technical details as possible while giving just a high view of the steps.

> Or is this a level of detail which you want to deal with in verification_steps.md only?

Im not sure what `verification_steps.md` will look like but generally at some lower level we will explain exactly what those verification steps are.

Reply:
I see. Do you think it would be beneficial to speak of "handhshake commitment" and "commitments to the plaintext". Currently one might get the impression that there is only a single commitment?

Reply:
My thinking was that if it says "openings" or "commitments", then the reader would wonder why there are multiple. He may feel that he may be misunderstanding something.

A single commitment should be sufficient for all normal use cases. The reader would probably think that we commit to the entirety of the data sent/received and later will selectively reveal portions of it.

The fact that currently our commitment protocol does not work like that (we dont commit to the entirety of data but we only commit to the portions which we intend to reveal) is a current design detail which can be changed later.

Reply:
alright, taking the perspective of someone new to the protocol, I agree that this is a useful simplification.